index.js

```diff
@@ -930,6 +930,75 @@
         }
     }
 
+    function applyUxxFilter() {
```

Comment:
This script has the same filter and adding this would ruin the view for everyone that installs both scripts.

https://greasyfork.org/en/scripts/467712-mz-tactics-selector
https://github.com/douglasdotv/tactics-selector